boards/arm/nucleo_l476rg/nucleo_l476rg.dts

```diff
@@ -173,3 +173,9 @@
 	pinctrl-names = "default";
 	status = "okay";
 };
+
+&dac1 {
+	pinctrl-0 = <&dac1_out1_pa4>;
```

Comment:
Please add chosen pin to docs also for this board.

Reply:
removed, not used anymore